zerver/lib/tex.py

```diff
@@ -0,0 +1,42 @@
+import os
+import subprocess
+from zerver.lib.str_utils import force_bytes
+from typing import Text
+
+def render_tex(tex, is_inline=True):
+    # type: (Text, bool) -> Text
+    """Render a TeX string into HTML using KaTeX
+
+    Returns the HTML string, or None if there was some error in the TeX syntax
+
+    Keyword arguments:
+    tex -- Text string with the TeX to render
+           Don't include delimiters ('$$', '\[ \]', etc.)
+    is_inline -- Boolean setting that indicates whether the render should be
+                 inline (i.e. for embedding it in text) or not. The latter
+                 will show the content centered, and in the "expanded" form
+                 (default True)
+    """
+    cd = os.path.dirname(os.path.realpath(__file__))
+
+    command = ['node']
+    if os.path.exists(os.path.normpath(cd + '/../../prod-static')):
+        # Running in production
+        command.append(os.path.normpath(os.path.join(
+            cd, '../../prod-static/serve/third/katex/cli.js')))
+    else:
+        # Running in a development environment
+        command.append(os.path.normpath(os.path.join(
+            cd, '../../static/third/katex/cli.js')))
```

Comment:
You should probably base these all off of `settings.DEPLOY_ROOT` rather than using realpath/normpath.